packages/avatar/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@availity/avatar",
+  "version": "0.1.0",
+  "author": "Tyson Warner <tyson.warner@availity.com>",
+  "description": "Availity User avatar component",
+  "main": "index.js",
+  "keywords": [
+    "react",
+    "availity",
+    "avatar"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Availity/availity-react.git"
+  },
+  "bugs": {
+    "url": "https://github.com/Availity/availity-react/issues"
+  },
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@availity/api-axios": "^5.4.0",
+    "@availity/api-core": "^5.4.0",
+    "@availity/localstorage-core": "^2.6.1",
+    "prop-types": "^15.5.8",
+    "react": "^16.8.3",
+    "react-dom": "^16.8.3"
```

Comment:
From that failed test looks like `axios` is missing from `devDeps`

Reply:
ec9883842bfdb01dc396cae2bde3c9d2f36dc4bb